src/v2/view-builder/views/email/BaseAuthenticatorEmailView.js

```diff
@@ -77,20 +77,29 @@ const Body = BaseFormWithPolling.extend(Object.assign(
       BaseForm.prototype.triggerAfterError.apply(this, arguments);
       this.stopPolling();
 
-      // Wait 1 min before polling again & do not show error message when encounter rate limit error
-      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests') ||
-        error.responseJSON?.errorCode === 'E0000047') {
+      // Polling needs to be stopped if it's a form error and session is invalid
+      if (error.responseJSON?.errorSummaryKeys?.includes('idx.session.expired')) {
+        return;
+      }
+
+      // Wait 1 min before polling again & do not show error message when polling encounter rate limit error
+      if (error.responseJSON?.errorSummaryKeys?.includes('tooManyRequests')
+          || error.responseJSON?.errorCode === 'E0000047') {
+
+        // if there is intent (ie. user submit OTP), resume polling
+        if (error.responseJSON?.errorIntent) {
+          this.startPolling(this.options.appState.get('dynamicRefreshInterval'));
+          return;
+        }
+
         setTimeout(() => {
```

Comment:
please add the comment here... Why these setTimeouts were added.. why specifically `60000` ?

Reply:
move comment explain the logic closer to the delay code